doc/docker/jenkinsnode/Dockerfile

```diff
@@ -0,0 +1,95 @@
+FROM debian:stretch
+
+ARG SWIGVERSION="3.0.11"
+
+# install dependencies
+RUN apt-get update && apt-get -y upgrade
+RUN apt-get -y install \
+  apt-utils \
+  curl \
+  build-essential \
+  clang \
+  autotools-dev \
+  automake \
```

Comment:
autotools-dev or automake are not needed

Reply:
copied from the a7, i'll try removing them to see if it still works afterwards